### PR TITLE
doc: use GAP, GAPDoc, LaTeX entities in more places

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: doc/Comments.xml [hunk 1]
```diff
@@ -327,10 +327,10 @@ Resets the current level to 0.
 <Index Key="@EndExample"><C>@EndExample</C></Index>
 <Heading><C>@BeginExample</C> and <C>@EndExample</C></Heading>
 <C>@BeginExample</C> inserts an example into the manual. The syntax for examples is different from
-GAPDocs example syntax in order to have a
-file that contains the example and is GAP readable. To achieve this, GAP commands are not preceded by a comment,
+&GAPDoc;'s example syntax in order to have a
+file that contains the example and is &GAP; readable. To achieve this, &GAP; commands are not preceded by a comment,
```
Comment (fingolfin) [inline doc/Comments.xml]: Of course that text still isn't really great... A casual reader won't know what "GAPDocs example syntax" even refers to. A precise reference to what is meant (presumably the `<Example>` element?), in the form of a `<Ref ... BookName="gapdoc"...` would be much better.

File: doc/Comments.xml [hunk 2]
```diff
@@ -387,7 +387,7 @@ The &AutoDoc; command <C>@ExampleSession</C> is an alias of <C>@BeginExampleSess
 <Index Key="@EndLog"><C>@EndLog</C></Index>
 <Heading><C>@BeginLog</C> and <C>@EndLog</C></Heading>
 Works just like the <C>@BeginExample</C> command, but the example
-will not be tested. See the GAPDoc manual for more information.
+will not be tested. See the &GAPDoc; manual for more information.
```
Comment (fingolfin) [inline doc/Comments.xml]: Here, too, we really should have an explicit  `<Ref ... BookName="gapdoc"...`

File: doc/Comments.xml [hunk 3]
```diff
@@ -396,7 +396,7 @@ The &AutoDoc; command <C>@Log</C> is an alias of <C>@BeginLog</C>.
 <Index Key="@EndLogSession"><C>@EndLogSession</C></Index>
 <Heading><C>@BeginLogSession</C> and <C>@EndLogSession</C></Heading>
 Works just like the <C>@BeginExampleSession</C> command, but the example
-will not be tested if manual examples are run. See the GAPDoc manual for more information.
+will not be tested if manual examples are run. See the &GAPDoc; manual for more information.
```
Comment (fingolfin) [inline doc/Comments.xml]: again:  `<Ref ... BookName="gapdoc"...`

**Comment by sebasguts** [comment]
Could you also update the expected manual, so the tests pass?

**Comment by fingolfin** [comment]
@sebasguts Oops! OK, PR fixed, Travis should pass now (we'll see shortly)

**Pull request merged**